opennebula/resource_opennebula_virtual_machine.go

```diff
@@ -123,10 +123,25 @@ func resourceOpennebulaVirtualMachine() *schema.Resource {
 				Computed:    true,
 				Description: "Current LCM state of the VM",
 			},
-			"cpu":      cpuSchema(),
-			"vcpu":     vcpuSchema(),
-			"memory":   memorySchema(),
-			"context":  contextSchema(),
+			"cpu":     cpuSchema(),
+			"vcpu":    vcpuSchema(),
+			"memory":  memorySchema(),
+			"context": contextSchema(),
+			"cpumodel": {
```

Comment:
you use it in shared schema.
It must be declared in shared schema to be also used by VM templates

Can you move this part in sharedSchema ?

Reply:
moved to shared_schemas.go